src/thesis.tex

```diff
@@ -731,7 +731,7 @@
 
     Following a straightforward instruction, everyone is able
     to create their own Docker image of the application.
-    Moreover, all of the configuration is present in one file,
+    Moreover, all of the configurations are present in one file,
```

Comment:
imo we rather mean a single configuration here but not sure

Reply:
fixed